css/style.css

```diff
@@ -92,24 +91,24 @@ button {
 .iconContainer img {
   height: 10rem;
 }
-.footer {
+footer {
   border-top: 1px solid #eee;
   text-align: center;
   margin: 4rem 0;
   padding: 0.5rem 0;
 }
-.footer .icons {
+footer .icons {
   display: flex;
   justify-content: center;
   align-items: center;
 }
-.footer .icon {
+footer .icon {
   border: 1px solid #ccc;
   border-radius: 50%;
   padding: 1rem;
   margin: 1rem;
 }
-.footer img {
+footer img {
   height: 1.5rem;
 }
 .join {
```

Comment:
This is good to remove the class from footer, as your site is likely to have only one footer